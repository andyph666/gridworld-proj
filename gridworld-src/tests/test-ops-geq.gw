int i = 0;
main{
	if(1 >= 2) {print("yes1");}
	if(1 >= 1) {print("yes2");}
	if(2 >= 1) {print("yes3");}
}