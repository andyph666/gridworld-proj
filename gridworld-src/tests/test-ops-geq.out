yes2
yes3